.vscode/settings.json

```diff
@@ -16,6 +16,16 @@
   "editor.codeActionsOnSave": {
     "source.fixAll.eslint": "explicit"
   },
+  "eslint.options": {
```

Comment:
I guess I need these for my vs code to run eslint on save...

but now i'm slightly confused b/c. ... how dis work before. 🤷

Reply:
huh that is weird 🤔 but that's what we have in ui-library 🤷